stdimage/management/commands/rendervariations.py

```diff
@@ -24,8 +27,16 @@ def add_arguments(self, parser):
                             default=False,
                             help='Replace existing files.')
 
+        parser.add_argument('--igrnore-missing',
+                            action='store_true',
+                            dest='igrnore_missing',
+                            default=False,
+                            help='Do not rise exception on missing file '
```

Comment:
`s/rise/raise/`

Reply:
How about something that does not include a negation:
```Ignore and skip missing file errors.```

Up in the help you might want to document, that the command will exit with non-zero exit code if a file is not found.